image/state.go

```diff
@@ -0,0 +1,65 @@
+package image
+
+import (
+	"sync"
+)
+
+var (
+	iSR = &imageStateMgr{
+		imgIDMap: make(map[ID]refCount),
+	}
```

Comment:
oh, I misunderstand, we should put it in `image.Store`, and reset it when docker starts up.

OK, Let's have a further look at the solution, then I will fix it.